permamodel/components/bmi_Ku.py

```diff
@@ -121,14 +120,13 @@ def initialize(self, config_file: str):
         self._start_time = 0.0
         self._end_time = self._model.number_of_years
         self._grid_shape = (
-            self._model.number_of_years,
             self._model.grid_shape[0],
             self._model.grid_shape[1],
         )
 
     def update(self):
         """Run the model for the current time step."""
-        self._model.run_one_step(self._current_time)
+        self._model.run_one_step(int(self._current_time))
```

Comment:
@mcflugen I worry this could be bad. Would you have any suggestions how to handle this better?